src/index.html

```diff
@@ -2,13 +2,43 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li><a href="#" class="is-active">Apple</a></li>
+          <li><a href="#" class="nav_link">Samsung</a></li>
+          <li><a href="#" class="nav_link">Smartphones</a></li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+          <li><a href="#" class="nav_link">Gadgets</a></li>
+          <li><a href="#" class="nav_link">Tablets</a></li>
+          <li><a href="#" class="nav_link">Photo</a></li>
+          <li><a href="#" class="nav_link">Video</a></li>
```

Comment:
Check it for 2 points in Checklist 
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md

Reply:
Sorry, but I don't get it here:  should I change like this?:
<li>
  <a href="#" class="nav_link is-active">Apple</a>
</li>